Cargo.toml

```diff
@@ -8,7 +8,6 @@ authors = ["Corey Farwell <coreyf@rwell.org>",
 license = "MIT/Apache-2.0"
 repository = "https://github.com/georust/rust-geojson"
 readme = "README.md"
-documentation = "https://georust.github.io/rust-geojson/"
```

Comment:
can we keep the `documentation` key here but have it point to the `docs.rs` documentation?